gh_deploy.sh

```diff
@@ -32,6 +32,8 @@ git remote add upstream "https://$GH_TOKEN@github.com/"${TRAVIS_REPO_SLUG}".git"
 git fetch upstream
 git reset upstream/gh-pages
 
+echo 'https://sched.eventyay.com' > CNAME
```

Comment:
Also, pick the value up from an environment variable called `GH_CNAME` instead. have added it to travis.